sources/packages/api/src-sql/AuthCodeTables/Remove-unused-types-roles.sql

```diff
@@ -0,0 +1,7 @@
+-- Remove unused types and roles. The related records on table 
+-- sims.institution_user_auth will be automatically removed.
+DELETE FROM
```

Comment:
If we are deleting the user-type e.g location-manager, what will happen to users who are assigned as location-manager already?

Reply:
The user type will be removed from the table `sims.institution_user_auth` also due to the cascading in place.

Reply:
Ah I see. 👍